plugins/org.eclipse.n4js/src/org/eclipse/n4js/organize/imports/ImportProvidedElement.java

```diff
@@ -25,10 +26,11 @@
 public class ImportProvidedElement {
 
 	/** name as used in script (can be alias) */
-	public String localname;
+	private final String localname;
```

Comment:
nice clean up! but please do the same clean up for fields `importSpec`, `tmodule`, `used` in same class.